### ISSUE TITLE
Pipe mode recovery does not work

### ISSUE THREAD

**Issue by greymd** [greymd/tmux-xpanes#113]
We can get recovery message when xpanes works with pipe on the out side of tmux session.
This is expected behavior, but ...

```
$ seq 3 | ./xpanes
xpanes:Warning: Attached session is required for 'Pipe mode'.
open terminal failed: not a terminal
xpanes:Recovery: Execute below command line to re-attach the new session.

tmux -S /home/docker/.cache/xpanes/socket.5840 attach-session -t xpanes-5840

```

`tmux -S /home/docker/.cache/xpanes/socket.5840 attach-session -t xpanes-5840` does not work because `/home/docker/.cache/xpanes/socket.5840` does not exists.

```
$ tmux -S /home/docker/.cache/xpanes/socket.5840 attach-session -t xpanes-5840
no sessions
```

This is caused by the previous changes released at [v3.1.1](https://github.com/greymd/tmux-xpanes/releases/tag/v3.1.1). Socket file is automatically deleted when the process exits.

It should be fixed next release.

**Comment by greymd** [comment]
This was not a caused by the previous release.
It looks it's an existing bug.

**Comment by greymd** [comment]
Fixed at [v4.0.0](https://github.com/greymd/tmux-xpanes/releases/tag/v4.0.0).
